cmd/where.go

```diff
@@ -0,0 +1,22 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var WhereCmd = &cobra.Command{
+	Use: "where",
+	Short: "Find where your cloud.charm.sh folder resides on your machine",
+	Long: paragraph("Find the absolute path to your charm keys, databases, etc."),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cc := initCharmClient()
+		path, err := cc.DataPath()
+		if err != nil {
+			return err
+		}
+		fmt.Fprint(cmd.OutOrStdout(), path)
```

Comment:
Looks good! I'd put a new line at the end of this though, so the prompt is on the next line after running `charm where`.

Reply:
Can do!